pkg/driver/nri_hooks.go

```diff
@@ -126,7 +124,13 @@ func (np *NetworkDriver) RunPodSandbox(ctx context.Context, pod *api.PodSandbox)
 			WithDriver(np.driverName).
 			WithPool(np.nodeName)
 
-		ifName := names.GetOriginalName(deviceName)
+		ifName, err := np.netdb.GetNetInterfaceName(deviceName)
+		if err != nil {
+			// Tip: We desired, tweak the code here to simply log
```

Comment:
This should be resolved prior to merge, do we want to support if a device has no corresponding network interface.